python/oneflow/framework/tensor.py

```diff
@@ -160,6 +160,10 @@ def _ne(self, other):
     return self.ne(other)
 
 
+def _contiguous(self):
+    return self
+
+
```

Comment:
```suggestion
def as_tensor(*args, **kwargs):
    # TODO: support construct tensor with sharing memory with data
    warnings.warn("This inferface is same with flow.tensor and not shared memory with data")
    return flow.tensor(*args, **kwargs)
```

要不你这个 PR 也顺手处理了吧，在 init 中导出一下就行

Reply:
好的，可以